images/tkg/docs/examples/adding_os_pkg_repos.md

```diff
@@ -0,0 +1,107 @@
+# Adding new OS packages and configuring the repositories or sources
+
+## Use case
+
+As a user I want to add new OS packages example `pkg1` and `pkg2` and also I want to configure ubuntu/photon sources/repositories to an internal mirror to build the node images in an air-gapped scenario. Configuring new sources/repositories is also useful when we want to install internally built software.
+
+## Customization
+
+OS packages and sources/repositories are configured through packer variables. To see the list of configurable packer variables provided by [kubernetes image builder][kubernetes-image-builder] please refer to this [page][customizations-doc].
+
+### Adding new packages
+
+[kubernetes image builder][kubernetes-image-builder] provides `extra_rpms` and `extra_debs` packer variables for Photon and Ubuntu respectively. Edit the [default-args.j2][default-args] to include the new packages.
```

Comment:
can we say 

To add new packages kubernetes image builder provides ...

To add new packages, add the packages to the packer variables in the file <filename> as shown below